### ISSUE TITLE
Deleting a comma after the last element in short syntax arrays

### ISSUE THREAD

**Issue by radziem** [FriendsOfPHP/PHP-CS-Fixer#1008]
I've noticed different result of deleting commas after the last element in arrays when I use short syntax array. First an example of an "old-style" arrays fixing:
```php
    $array53 = array(
      array("1", "2", "3", ),
      array("4", "5", "6", ),
      array("7", "8", "9", ),
    );
```
and the expected result:
```php
    $array53 = array(
      array("1", "2", "3"),
      array("4", "5", "6"),
      array("7", "8", "9"),
    );
```
It looks good, but the example of short syntax array:
```php
    $array54 = [
      ["1", "2", "3", ],
      ["4", "5", "6", ],
      ["7", "8", "9", ],
    ];
```
gives a little different result:
```php
    $array54 = [
      ["1", "2", "3"],
      ["4", "5", "6", ],
      ["7", "8", "9", ],
    ];
```

**Comment by keradus** [comment]
On both 1.4.2 and 1.5-DEV version
```diff
diff --git a/test.php b/test.php
index 112911c..8714b6a 100644
--- a/1/x.php
+++ b/1/x.php
@@ -1,13 +1,13 @@
 <?php
 
 $array53 = array(
-    array("1", "2", "3", ),
-    array("4", "5", "6", ),
-    array("7", "8", "9", ),
+    array("1", "2", "3"),
+    array("4", "5", "6"),
+    array("7", "8", "9"),
 );
 
 $array54 = [
-    ["1", "2", "3", ],
-    ["4", "5", "6", ],
-    ["7", "8", "9", ],
+    ["1", "2", "3"],
+    ["4", "5", "6"],
+    ["7", "8", "9"],
 ];
```

Which version are you using?
Do you have any custom config?

**Comment by keradus** [comment]
IMO you are using <1.4.2 version. It should be already fixed by #994 (merged into v1.4.2).
Please update to 1.4.2@stable or 1.5@dev and check it again.

**Comment by radziem** [comment]
You're right, I used version 1.4.1.
After update everything is fixed as expected.
